src/components/ToolMenu/Draw/StylingDrawerButton/index.tsx

```diff
@@ -24,9 +25,14 @@ export const StylingButton: React.FC = (): JSX.Element => {
   const {
     t
   } = useTranslation();
+  const drawerVisibility = useAppSelector(state => state.stylingDrawerVisibility);
 
   const showDrawer = () => {
-    dispatch(setStylingDrawerVisibility(true));
+    if (!drawerVisibility){
+      dispatch(setStylingDrawerVisibility(true));
+    } else {
+      dispatch(setStylingDrawerVisibility(false));
+    }
```

Comment:
can be simplified to:
```suggestion
  dispatch(setStylingDrawerVisibility(!drawerVisibility));
```

Reply:
done in: [2c93f63](https://github.com/terrestris/shogun-gis-client/pull/1562/commits/2c93f63d742d1e0b3e3778b754180dccbc371a59)